extensions/2.0/Khronos/KHR_texture_float_bc6h/README.md

```diff
@@ -0,0 +1,219 @@
+﻿
+﻿# KHR_texture_float_bc6h
+
+## Contributors
+
+* Rickard Sahlin, mailto:rickard.sahlin@ikea.com  
+* Alexey Knyazev [@lexaknyazev](https://github.com/lexaknyazev)  
+* Don McCurdy, Google, [@donrmccurdy](https://twitter.com/donrmccurdy)  
+* Norbert Nopper, mailto:nopper@ux3d.io  
+
+Copyright (C) 2021 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.  
+
+## Overview
+
+This extension adds the ability to specify textures using KTX v2 images with compressed float format as defined by VK_FORMAT_BC6H_UFLOAT_BLOCK.  
+It allows specification of a source texture with increased dynamic range to be used.  
+This source texture shall be used in shader (BRDF) calculations in a way that retains the increased range.  
+However, this extension does not define the way pixel data is written to framebuffer or how swapchain is presented using the increased range.  
+
+When the extension is used, it's allowed to use value `image/ktx2` for the `mimeType` property of images that are referenced by the `source` property of `KHR_texture_float_bc6h` texture extension object.
+
+At runtime, engines are expected to check hardware support for VK_FORMAT_BC6H_UFLOAT_BLOCK, if not present the extension is not supported.  
+This means that it is not allowed to transcode the texture data to some other format.   
```

Comment:
glTF does not prescribe runtime behaviour so this statement cannot be included. If there's an application-specific use-case, this should be a vendor extension.

Reply:
Removed wording and added conformance section.